tests/FakeItEasy.Specs/ConfigurationSpecs.cs

```diff
@@ -697,6 +697,7 @@ public virtual void DoSomething()
 
             public void DoSomethingNonVirtual()
             {
+                this.WasCalled = true;

```

Comment:
I must be missing something. It's a line of code, and nothing else has changed in the file.